python/core/utils/scratch_tests.py

```
@@ -1,66 +1,72 @@
-            raise ValueError(f"Tensor {name} not found in model graph.")
+            msg = f"Tensor {name} not found in model graph."
+            raise ValueError(msg)
 
         elem_type = value_info.type.tensor_type.elem_type
         shape = [dim.dim_value for dim in value_info.type.tensor_type.shape.dim]
         new_output = helper.make_tensor_value_info(name, elem_type, shape)
         graph.output.append(new_output)
+
     for output in graph.output:
-        print(output)
+        print(output)  # noqa: T201
```

Ruff (0.14.7)

Unused `noqa` directive (non-enabled: `T201`)

Remove unused `noqa` directive

(RUF100)